registry/quay.go

```diff
@@ -0,0 +1,74 @@
+package registry
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/coreos/go-quay/quay"
+	"github.com/coreos/go-quay/quay/tag"
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/strfmt"
+)
+
+const (
+	quayTokenEnvVarName = "QUAY_AUTH_TOKEN"
+)
+
+// QuayRegistry includes a Client, Auth and helper methods for interacting
+// with the quay.io api
```

Comment:
@vdice it's probably a good idea to mention in these docs that `QuayRegistry` implements the `Registry` interface. Same with the docker hub implementation above